exampleCourse/questions/element/matching/question.html

```diff
@@ -150,3 +150,25 @@
         </pl-matching>
     </div>
 </div>
+
+<div class="card my-2">
+    <div class="card-body">
+        <pl-question-panel>
+            <p>
+                Example 8: This example uses the `options-placement="bottom"` argument. This is useful for questions with long statements and options, which may wrap over multiple lines or cause formatting issues in the default "right" placement.
+            </p>
+        </pl-question-panel>
+            
+        <pl-matching answers-name="ex8" options-placement="bottom">
+            <pl-statement match="The theory of relativity">Which theory revolutionized our understanding of space, time, and gravity?</pl-statement>
+            <pl-statement match="Quantum mechanics">What is a branch of physics that challenges classical intuition and underpins technologies like quantum computing?</pl-statement>
+            <pl-statement match="Artificial intelligence">What field of computer science encompasses machine learning, natural language processing, and computer vision?</pl-statement>
+            <pl-statement match="Genetic engineering">What biotechnology technique offers potential in treating genetic disorders and enhancing crop yields?</pl-statement>
+            
+            <pl-option>Traditional computing</pl-option>
+            <pl-option>Data Science</pl-option>
+            <pl-option>OPERA faster-than-light neutrino anomaly</pl-option>
+            <pl-option>Genome Editing</pl-option>
```

Comment:
Nit, consistent casing with the correct answers:

```suggestion
            <pl-option>Data science</pl-option>
            <pl-option>OPERA faster-than-light neutrino anomaly</pl-option>
            <pl-option>Genome editing</pl-option>
```

I'm no biologist, but couldn't "genome editing" be considered a subset of "genetic engineering" that itself could "offer potential in treating genetic disorders and enhancing crop yields"? I see lots of references to CRISPR in relation to both GMOs and medical therapies.